web-frame/src/app-shared/core/initializers/app-state-initializer.service.ts

```diff
@@ -36,6 +36,10 @@ export class AppStateInitializerService {
   private logoutAndReloadApplication(): Observable<never> {
     return this.accountService.logout().pipe(
       mergeMap((): Observable<never> => {
+        // TODO: Fix possible race condition and use translated messages.
```

Comment:
Create follow up task and update TODO with the id.

Reply:
Actually, I just realized that the final version didn't have a race condition so I've update the code to use the translation system.